lib/es-models/base.js

```diff
@@ -0,0 +1,7 @@
+class EsBase {
+  updatedAt () {
```

Comment:
I included an updatedAt function in the indexing function. Not sure if it's better off here or not. I think there will have to be some work to reconcile any duplicated or convoluted logic some time down the line. What do you think, @nonword ?

Reply:
We want an `updatedAt` property on the bib, but not on each item or holding (because the current meaning of this `updatedAt` is "what time was this indexed?", so all `updatedAt` values would be the same if we added it to items and holdings). So I think it might need to go in `EsBib` only. Otherwise, the indexing will fail when it attempts to add `updatedAt` properties to items and holdings.

Reply:
done!